recipe/bld.bat

```diff
@@ -0,0 +1,31 @@
+mkdir %SRC_DIR%\build
+cd %SRC_DIR%\build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -D NETCDF_INCLUDE=%LIBRARY_INC% ^
+      -D NETCDF_LIBRARY=%LIBRARY_LIB%\netcdf.lib ^
+      -D HDF5_LIBRARY=%LIBRARY_LIB%\libhdf5.lib ^
+      -D HDF5_HL_LIBRARY=%LIBRARY_LIB%\libhdf5_hl.lib ^
+      -D GSL_INCLUDE=%LIBRARY_INC% ^
+      -D GSL_LIBRARY=%LIBRARY_LIB%\gsl.lib ^
+      -D GSL_CBLAS_LIBRARY=%LIBRARY_LIB%\gslcblas.lib ^
+      -D ZLIB_LIBRARY=%LIBRARY_LIB%\zlib.lib ^
+      -D UDUNITS2_INCLUDE=%LIBRARY_LIB% ^
+      -D UDUNITS2_LIBRARY=%LIBRARY_LIB%\udunits2.lib ^
+      -D EXPAT_LIBRARY=%LIBRARY_LIB%\expat.lib ^
+      -D CURL_LIBRARY=%LIBRARY_LIB%\libcurl.lib ^
+      -D LIB_ANTLR=%LIBRARY_LIB%\antlr.lib ^
+      -D HEADER_ANTLR=%LIBRARY_INC% ^
```

Comment:
@pedro-vicente is this correct? I got [this warning](https://ci.appveyor.com/project/conda-forge/nco-feedstock/build/1.0.24#L721) on Appveyor:

```
CMake Warning:
  Manually-specified variables were not used by the project:
    GSL_CBLAS_LIBRARY
    GSL_INCLUDE
    GSL_LIBRARY
    HEADER_ANTLR
    LIB_ANTLR
    ZLIB_LIBRARY
```

later on the library is found but the headers are now, see https://ci.appveyor.com/project/conda-forge/nco-feedstock/build/1.0.24#L693

The conda package for `antlr` is [here](https://anaconda.org/conda-forge/antlr/2.7.7/download/win-64/antlr-2.7.7-vc14_4.tar.bz2) and the headers are in the `Library\include` directory. [Note that I copied them manually](https://github.com/conda-forge/antlr-feedstock/blob/master/recipe/bld.bat#L14) b/c the build process does not place them there, so maybe I missed some?

Reply:
If you compare the entry in CMakeLists.txt

```
find_path(ANTLR_INCLUDE antlr/AST.hpp ${find_opt})
```

with the example in /cmake/build.bat

```
-DANTLR_INCLUDE:PATH=I:/antlr-2.7.7/lib/cpp
````

in this case, it means that this file must exist

```
I:/antlr-2.7.7/lib/cpp/antlr/AST.hpp
```

Reply:
I sent a PR to antlr-feedstock to put the headers in a `antlr` subfolder. Let's see how that goes.
(Thanks @isuruf for the tip!)